dnf/cli/cli.py

```diff
@@ -883,6 +883,9 @@ def _read_conf_file(self, releasever=None):
             releasever = dnf.rpm.detect_releasever(conf.installroot)
         elif releasever == '/':
             releasever = dnf.rpm.detect_releasever(releasever)
+        if releasever is None:
+            logger.warning(
+                _('Unable to detect "releasever". It can be set by "--releasever" option'))
```

Comment:
I would change the message to something like this: 'Unable to detect release version (add '--releasever' to command line)'. Optionally '... command line to specify release version)'.